provd/devices/config.py

```diff
@@ -92,20 +92,20 @@
   If the device only support HTTP yet this value is not defined, an Exception
   SHOULD be raised.
 
-http_base_url [optional]
+http_base_url [mandatory]
   Used in place of the ip and http_port to be able to define the URL at which
   the provisioning server is accessible from. This allows to set the provisioning
   server behind a reverse proxy.
   If you are using a `url_key` strategy for provisioning authentication (provisioning
   key), then you MUST use this setting as the key is appended to this value.
-  This option has precedence over the ip + http_port combination.
+  This option has precedence over the deprecated ip + http_port combination.
 
-tftp_port [mandatory if http_port is not defined]
+tftp_port [mandatory if http_base_url is not defined]
```

Comment:
I don't see any logic related to this. Did you plan on implementing the logic or is that a documentation mistake?